tools/tuf/README.md

```diff
@@ -294,6 +294,32 @@ TIMESTAMP_PASSPHRASE_1PASSWORD_PATH="Private/UPDATES TIMESTAMP/password" \
 ACTION=release-to-production ./tools/tuf/releaser.sh
 ```
 
+### Doing a patch release of fleetd
+
+Patch releases follow the same process as releasing a minor version, except instead of checking out the `main` branch of fleet locally, you check out a patch branch of fleet, e.g.:
+
+```
+git checkout rc-minor-fleet-v.4.65.1
+```
+
+As always, the `VERSION` env var used when running `releaser.sh` should match the version of the fleetd release, e.g.
```

Comment:
Folks releasing can take a look at  the latest released version here: https://github.com/fleetdm/fleet/blob/main/orbit/TUF.md.